dashboard/config/locales/en.yml

```diff
@@ -1424,6 +1424,9 @@ en:
       form_instructions_2: "Full guides for obtaining your LMS Client ID, using LMS/LTI integrations, and our list of supported Learning Management Systems <a href='https://code.org/contact'>can be found here.</a>"
       form_instructions_3: "Not seeing your LMS in our list, or running into issues with your installation? Check out our <a href=''>guides</a>, or <a href='https://code.org/contact'>contact us</a> for additional help"
       guide_instructions: "Full guides for installing and our LTI integration <a href=''>can be found here.</a> Running into issues? Feel free to <a href=''>contact us</a> for additional support."
+      mailer_confirmation_subject: "Code.org has completed your LMS registration request"
+      mailer_confirmation_body: "Great news! We’ve registered your LMS with Code.org. You should now be able to use our Code.org LMS integrations. Please feel free to review & share our <a href=''>LMS integration guides</a>, or feel free to browse our <a href=''>course offerings</a>."
```

Comment:
Will populate the links with valid URL's in a follow-up PR once Ali supplies them.